framework/src/bundle/BundleHooks.h

```diff
@@ -23,7 +23,7 @@
 #ifndef CPPMICROSERVICES_BUNDLEHOOKS_P_H
```

Comment:
I made the header include guards changes to reflect the filenames, so I think removing the (and remove the `_P` suffix makes sense. However, I checked to see if there are headers that share the same name in different locations, and I found one public and one private version of "ServiceListenerHook.h". I don't know what the right move forward would be; perhaps we can just leave the current header include guards as is for now.